### ISSUE TITLE
Upgrade Accumulo version to 1.9.3

### ISSUE THREAD

**Issue by t11947** [gchq/gaffer-tools#827]
In line with Gaffer issue [#2249](https://github.com/gchq/Gaffer/issues/2249), the accumulo version needs to be upgraded in Gaffer tools. Upgrade any deprecated functions from the Accumulo API. 
Backwards compatibility should be ensured such that anyone still using older versions of Accumulo can still use the new version of Gaffer without having to upgrade their Accumulo installation.

Info about 1.9.3 here: https://accumulo.apache.org/release/accumulo-1.9.3/

**Comment by phosphorbronze50** [comment]
Issue:
When gaffer-tools builds for accumulo 1.9.3 I get a compile error with:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile(default-compile) on project performance-testing-aws:Compilation failure[ERROR] .../performance-testing/performance-testing-aws/src/main/java/uk/gov///performancetesting/aws/PublishAccumuloMetricsToCloudWatch.java:[100,57] error: incompatible types: Map<String,String> cannot be converted to SortedMap<String,String>

Explanation:
The AWS CloudWatch reporter obtains information from the Accumulo system - a list of tables in the database. This information used to be returned (ver. 1.8.1) as a SortedMap<String, String> but in the newer version of Accumulo (ver. 1.9.3) this has been changed to a Map<String, String> (SortedMap's superclass).

Solution:
I have changed the code so the local attribute is now a Map<String, String> and this cures the compilation error. The attribute is private and only used in this immediate area of the code.

The change compiles OK when used with either of the above mentioned accumulo versions.
